integration/core/test_cli.py

```diff
@@ -322,21 +322,30 @@ def test_snapshot_rm(bin, controller_client, replica_client, replica_client2):
     assert v.replicaCount == 2
 
     cmd = [bin, 'snapshot', 'create']
-    subprocess.check_call(cmd)
-    output = subprocess.check_output(cmd).strip()
 
+    # first snapshot
+    output1 = subprocess.check_output(cmd).strip()
+    chain = replica_client.list_replica()[0].chain
+    assert len(chain) == 2
+    assert chain[0] == 'volume-head-001.img'
+    assert chain[1] == 'volume-snap-{}.img'.format(output1)
+
+    # second snapshot
+    output2 = subprocess.check_output(cmd).strip()
     chain = replica_client.list_replica()[0].chain
     assert len(chain) == 3
     assert chain[0] == 'volume-head-002.img'
-    assert chain[1] == 'volume-snap-{}.img'.format(output)
+    assert chain[1] == 'volume-snap-{}.img'.format(output2)
+    assert chain[2] == 'volume-snap-{}.img'.format(output1)
 
-    cmd = [bin, 'snapshot', 'rm', output]
+    # remove the first snapshot(empty), it will fold second snapshot(empty)
+    # to the first snapshot(empty) and rename it to second snapshot
+    cmd = [bin, 'snapshot', 'rm', output1]
     subprocess.check_call(cmd)
-
     new_chain = replica_client.list_replica()[0].chain
     assert len(new_chain) == 2
     assert chain[0] == new_chain[0]
-    assert chain[2] == new_chain[1]
+    assert chain[1] == new_chain[1]
```

Comment:
You need to add a new test case, not replacing the old one.

The new tests should have "more" coverage than old one, not "different" coverage.